app/views/console/users/team.phtml

```diff
@@ -111,14 +111,14 @@
                                                 <button class="danger">Remove</button>
                                             </form>
 
-                                            <img src="" data-ls-attrs="src={{member|avatar}}" data-size="200" alt="User Avatar" class="avatar pull-start margin-end" loading="lazy" width="60" height="60" />
```

Comment:
Not sure about this change. Why from `member` to `member.userName` Was `member` previously object, ID, email or name?

Reply:
https://github.com/appwrite/appwrite/blob/70efbf1fb9c3277043ef0255ba320b02beee3bbe/public/scripts/filters.js#L8

We have a fallback to the name property 👍🏻